monai/networks/utils.py

```diff
@@ -707,16 +707,16 @@ def convert_to_torchscript(
         rtol: the relative tolerance when comparing the outputs of PyTorch model and TorchScript model.
         atol: the absolute tolerance when comparing the outputs of PyTorch model and TorchScript model.
         use_trace: whether to use `torch.jit.trace` to export the torchscript model.
-        kwargs: other arguments except `obj` for `torch.jit.script()` to convert model, for more details:
-            https://pytorch.org/docs/master/generated/torch.jit.script.html.
+        kwargs: other arguments except `obj` for `torch.jit.script()` or `torch.jit.trace()` to convert model,
```

Comment:
suggest to write as:

or `torch.jit.trace()` (if `use_trace` is True) to convert model